app/src/components/notifications/NotificationToast.vue

```diff
@@ -54,37 +55,41 @@ switch (props.notification.state) {
         v-if="show && notification.type == 'toast'"
         class="pointer-events-auto w-full max-w-sm overflow-hidden rounded-lg bg-white shadow-lg ring-1 ring-black ring-opacity-5"
     >
-        <div class="p-4">
-            <div class="flex items-start">
-                <div class="flex-shrink-0">
-                    <component
-                        v-if="icon"
-                        :is="icon"
-                        class="h-6 w-6"
-                        :class="textColor"
-                        aria-hidden="true"
-                    />
-                </div>
-                <div class="ml-3 w-0 flex-1 pt-0.5">
-                    <p class="text-sm font-medium text-zinc-900">{{ notification.title }}</p>
-                    <p class="mt-1 text-sm text-zinc-500">
-                        {{ notification.description }}
-                    </p>
-                </div>
-                <div class="ml-4 flex flex-shrink-0">
-                    <button
-                        type="button"
-                        @click="
-                            notification.id ? removeNotification(notification.id) : (show = false)
-                        "
-                        class="inline-flex rounded-md bg-white text-zinc-400 hover:text-zinc-500 focus:outline-none focus:ring-2 focus:ring-indigo-500 focus:ring-offset-2"
-                        data-test="toast"
-                    >
-                        <span class="sr-only">Close</span>
-                        <XMarkIcon class="h-5 w-5" aria-hidden="true" />
-                    </button>
+        <component :is="notification.routerLink ? RouterLink : 'div'" :to="notification.routerLink">
```

Comment:
If it is a router link, it should close the notification when clicked. I think adding a click handler here would solve that (I think it was there before but was removed in a commit).